imblearn/under_sampling/_prototype_selection/_edited_nearest_neighbours.py

```diff
@@ -120,7 +120,7 @@ def __init__(
     def _validate_estimator(self):
         """Validate the estimator created in the ENN."""
         self.nn_ = check_neighbors_object(
-            "n_neighbors", self.n_neighbors, additional_neighbor=1
+            "n_neighbors", self.n_neighbors, additional_neighbor=0
```

Comment:
the current version of the code adds 1 to the number of neighbours entered by the user if the user passes an integer. So the docs say that the default is a KNN with 3 neighbours, but it is actually training a KNN with 4 neighbours. This fixes the behaviour